Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -301,6 +302,45 @@
     return dictionaryPopupInfo;
 }
 
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+void WebPage::getTextIndicatorForID(const WTF::UUID uuid, CompletionHandler<void(std::optional<WebCore::TextIndicatorData>&&)>&& completionHander)
```

Comment:
I think maybe all of this logic should be moved to the unified text replacement controller